GUI/QtQuick/qml/ConfirmPassword.qml

```diff
@@ -105,18 +109,18 @@ ColumnLayout  {
                 confirm_password.isValid = false
             }
         }
-
-        onClicked: {
-            click_enter()
-        }
     }
 
     Keys.onEnterPressed: {
-        confirm_but.click_enter()
+         confirm_but.click_enter()
```

Comment:
indentation is too much, IMO
we normally use 3 spaces, 4 is also good, but 5 is excessive

Reply:
Ok agree